frontend/src/__tests__/DeleteCommentOrPosts.test.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
good idea using the same component to delete both posts and comments - reusable :)